pynumaflow/tests/function/test_async_server_err.py

```diff
@@ -0,0 +1,149 @@
+import asyncio
```

Comment:
This file is basically a copy of `test_async_server.py`. I added a test case to cover reduce handler raising RuntimeError scenario. To avoid port address conflict, I am using a new port number `50052` in this test.